lib/workers/repository/process/deprecated.js

```diff
@@ -53,5 +56,20 @@ async function raiseDeprecationWarnings(config, packageFiles) {
         await platform.ensureIssue(issueTitle, issueBody, ensureOnce);
       }
     }
+    logger.debug(
+      'Closing existing depricated package issues missing in current deprecatedPackages'
```

Comment:
This should say "Checking for" not "closing". Otherwise it implies that you are closing when most times you probably are not. Also depricated => deprecated